Utils/CategoryMerchandising.php

```diff
@@ -46,19 +44,4 @@ class CategoryMerchandising
     const DISPATCH_EVENT_KEY_RESULT = 'result';
     const DISPATCH_EVENT_KEY_LIMIT = 'limit';
```

Comment:
Move these constants to other classes, like Pre/PostRequestAction and delete this class